components/expenses/PayoutBankInformationForm.js

```diff
@@ -213,26 +203,44 @@ const DetailsForm = ({ disabled, getFieldName, formik, host, currency }) => {
     return <P>{error.message}</P>;
   }
 
-  const [requiredFields] = data.host.transferwise.requiredFields;
-  const [addressFields, otherFields] = partition(requiredFields.fields, f =>
+  const transactionTypeValues = data.host.transferwise.requiredFields.map(rf => ({ label: rf.title, value: rf.type }));
+  const requiredFields = data.host.transferwise.requiredFields.find(
+    rf => rf.type == get(formik.values, getFieldName(`data.type`)),
+  );
+  const [addressFields, otherFields] = partition(requiredFields?.fields, f =>
     f.group.every(g => g.key.includes('address.')),
   );
 
   return (
     <Flex flexDirection="column">
-      <Box mt={2} flex="1">
-        <P fontSize="16px" fontWeight="bold">
-          {requiredFields.title}
-        </P>
-      </Box>
+      <Field name={getFieldName('data.type')}>
+        {({ field }) => (
+          <StyledInputField name={field.name} label="Transaction Type" labelFontSize="13px" mt={3} mb={2}>
+            {({ id }) => (
+              <StyledSelect
+                inputId={id}
+                name={field.name}
+                onChange={({ value }) => {
+                  const { type, currency } = get(formik.values, getFieldName(`data`));
+                  formik.setFieldValue(getFieldName('data'), { type, currency });
+                  formik.setFieldValue(field.name, value);
+                }}
+                options={transactionTypeValues}
+                value={transactionTypeValues.find(c => c.value === requiredFields?.type) || null}
```

Comment:
Same thing here, I think it would be better to explicitly write `currency` rather than `c`